packages/core/addon/components/navi-visualizations/line-chart.js

```diff
@@ -215,6 +219,28 @@ export default Component.extend({
     };
   }),
 
+  /**
+   * Gets current options and decides c3 chart type
+   * @private
+   * @method
+   * @returns {String} - c3 chart type to determine line behavior
+   */
+  _lineType() {
```

Comment:
Can this be a cp called `_c3ChartType`?

Reply:
computed property caused a problem with recursive call.  Also since it's used as a utility in a computed property that has the right dependencies, we don't need this to be a cp.

Reply:
What is recursive?

Reply:
Looks like `dataConfig` is cache busting when `seriesConfig` change when `options` changes. Can we make this a direct dependency instead of an indirect dependency?